packages/coding-components/templates/angular/package.json

```diff
@@ -1,29 +1,25 @@
 {
   "name": "coding-components-angular-template",
   "version": "0.0.0",
+  "private": true,
   "scripts": {
     "ng": "ng",
     "start": "ng serve -o",
```

Comment:
Consider switching out with ` "start": "ng serve --force-esbuild",` it's much faster 😄